CHANGELOG.md

```diff
@@ -2,6 +2,19 @@
 
 ## Unreleased
 
+### Features
+
+- Add the following functions on `@sentry/capacitor` ([#666](https://github.com/getsentry/sentry-capacitor/pull/666))
+
+* flush.
+* getActiveSpan.
+* spanToJSON.
+* spanIsSampled.
+* setMeasurement.
+* getGlobalScope.
+* getClient.
+* setCurrentClient.
+
```

Comment:
Listing the function definitely make sense, I would just move it under the related entry. 

What do you think?

```sugestion
### Features

- Add the following functions to `@sentry/capacitor` ([#666](https://github.com/getsentry/sentry-capacitor/pull/666))
  - `flush`, `getActiveSpan`, `spanToJSON`, `spanIsSampled`, `setMeasurement`, `getGlobalScope`, `getClient` and `setCurrentClient`.

```